app.py

```diff
@@ -15,6 +16,7 @@
 # User Registration
 @app.route('/register', methods=['GET', 'POST'])
```

Comment:
- [ ] A person who is logged in should not be able to access this route, add a route guard